docs/model-vocabulary.md

```diff
@@ -0,0 +1,94 @@
+---
+id: model-vocabulary
+title: Vocabulary
+---
+
+The Vocabulary module for Concerto optionally allows human-readable labels (Terms) to be associated with model elements. Terms are stored within a locale specific vocabulary YAML file associated with a Concerto namespace.
+
+For example, a Concerto model that defines an enumeration with the values `RED`, `GREEN`, `BLUE` can be associated with an English vocabulary with the terms "Red", "Green", "Blue" and a French Vocabulary with terms "Rouge", "Vert", "Bleue".
+
+The VocabularyManager class manages access to a set of Vocabulary files, and includes logic to retrieve the most appropriate term for a requested locale.
```

Comment:
My only thought is maybe VocabularyManager should be code formatted (`VocabularyManager`)